pkgs/applications/networking/instant-messengers/telegram/kotatogram-desktop/Use-system-font-by-default.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/Telegram/SourceFiles/settings.cpp b/Telegram/SourceFiles/settings.cpp
+index 4cde4a658..71e6954f3 100644
+--- a/Telegram/SourceFiles/settings.cpp
++++ b/Telegram/SourceFiles/settings.cpp
+@@ -7,6 +7,7 @@ https://github.com/telegramdesktop/tdesktop/blob/master/LEGAL
+ */
+ #include "settings.h"
+ 
++#include "base/platform/base_platform_info.h"
+ #include "ui/emoji_config.h"
+ 
+ namespace {
+@@ -209,7 +210,7 @@ rpl::producer<> UpdatedRecentEmoji() {
+ 
+ QString gMainFont, gSemiboldFont, gMonospaceFont;
+ bool gSemiboldFontIsBold = false;
+-bool gUseSystemFont = false;
++bool gUseSystemFont = Platform::IsLinux();
```

Comment:
This feels useful to the kotatogram-desktop repository, A PR should probably be made against it

Reply:
This setting introduced by other patch, I just separate it for convenience

Reply:
I noticed. But I feel like all the patches should be made into a PR for the kotatogram package, as that's probably what most Linux users want.

What I'm trying to say is, doing patches to the nix packages works, but if it's able to benefit the greater community, then the patches should go into master on the related project

Reply:
They will, but at that point there are only this way